testers/testers/sql/bin/install.sh

```diff
@@ -7,16 +7,30 @@ install_packages() {
     sudo apt-get install python3 postgresql
 }
 
+random_pwd() {
+    date +%s%N | sha256sum | base64 | head -c 32
+}
+
 create_db_and_users() {
     echo "[SQL-INSTALL] Creating databases and users"
     echo "[SQL-INSTALL] Creating oracle user '${ORACLEUSER}' with database '${ORACLEDB}'"
     sudo -u postgres psql <<-EOF
 		DROP DATABASE IF EXISTS ${ORACLEDB};
 		DROP ROLE IF EXISTS ${ORACLEUSER};
 	EOF
-    echo "[SQL-INSTALL] Create password for oracle user '${ORACLEUSER}'"
-    sudo -u postgres createuser -P ${ORACLEUSER} # secure, password is not logged
+
+    if id -u ${ORACLEUSER} &> /dev/null; then
+        sudo -u "${ORACLEUSER}" -- bash -c "touch ${HOME}/.pgpass && chmod 600 ${HOME}/.pgpass"
```

Comment:
HOME here will be expanded before doing sudo, it's going to be the home of the caller, not oracleuser